src/pages/admin-ui-sdk/publish.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Prepare your app for production
+description: Learn how to prepare your app for non-local testing and for production.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# Prepare your app for production
+
+After you have tested your app locally, you are ready to publish your app to your Production environment. When you have accomplished this, your app is available in the Adobe App Registry. Adobe Commerce will then be able fetch the information it needs to augment the Admin.
+
+During the testing phase, an administrator of your enterprise organization performs the approvals. At this point, your app is not public. When you are ready to make the app available publicly, submit the app to the Adobe Review team instead of your admin.
```

Comment:
Let's remove this line - the Adobe Review team is actually never involved in the private distribution process that is required for the Admin UI SDK. The existing App Builder docs are very confusing and somewhat incorrect. In the current flow, an app is always reviewed by the IMS org admin and is distributed privately (only accessible to the IMS organization)